ios/Podfile

```diff
@@ -8,6 +8,8 @@ target 'Mobile' do
 
   use_react_native!(path: config[:reactNativePath], hermes_enabled: true)
 
+  pod 'BVLinearGradient', :path => '../node_modules/react-native-linear-gradient'
```

Comment:
With auto-linking, we shouldn't need to add the pod manually to each Android or iOS - you should be able to remove any changes from the `ios` and `android` folders. See https://github.com/react-native-linear-gradient/react-native-linear-gradient#installation for more information!

Reply:
I don't totally get it. These changes in ios and android we made through the auto-linking. I used the auto-linking command `react-native link react-native-linear-gradient`.

Reply:
Ah, that's actually automatic manual linking - auto-linking is just when React Native is able to resolve it itself in the podfile without making any code changes - all you have to do is install the dependency via npm and run `pod install` (or in our case, `npm i` as we have a `postinstall` script that does the `pod install`). Here's some more information: https://github.com/react-native-community/cli/blob/master/docs/autolinking.md

Reply:
> Autolinking is a replacement for react-native link. If you have been using React Native before version 0.60, please unlink native dependencies if you have any from a previous install.

😄